### ISSUE TITLE
DSA555 Course notes

### ISSUE THREAD

**Issue by joulecpa** [cathyatseneca/dsa555-w15#5]
The gitbook that is used as our course notes for some reason has become the GAM536 course notes.
![capture](https://cloud.githubusercontent.com/assets/10556836/7014281/d5ef90e2-dc90-11e4-89be-7fec328c2570.PNG)

**Comment by cathyatseneca** [comment]
yeah sorry I accidentally pushed up my modeling notes to the wrong repo.  Its fixed now.  Please let me know if it is not...

**Comment by joulecpa** [comment]
Looks fine now thanks.
